report_templates/index.Rmd.in

```diff
@@ -407,6 +407,42 @@ if( fetch_refgen_success )
 
 ```
 
+
+```{r plot_methylation_near_TSSs, eval = fetch_refgen_success }
+TSS_plotlength  <- params$TSS_plotlength
+
+ymin = 0
+ymax = 100
+
+gene_features   <- readTranscriptFeatures(refGenes_bedfile)
+TSS_exact       <- gene_features$TSSes
+TSS_proximal    <- resize(TSS_exact, width=TSS_plotlength, fix='center')
+
+sample_methylome_gr          <-  as(methRaw, "GRanges")
```

Comment:
You already exported the methylation signal to bigwig, so just use this with the ScoreMatrix function.

Reply:
This should decrease your memory footprint.

Reply:
The issue for this is posted here. https://github.com/BIMSBbioinfo/pigx_bsseq/issues/119
 If either of you want to address it, feel free to, but I'm moving on to more time-pressing issues.

Reply:
> This should decrease your memory footprint.

@alexg9010 , can you elaborate on this. What exactly are you proposing to reduce the memory footprint. I'm hitting memory errors now on the cluster submissions.